EIPS/eip-7093.md

```diff
@@ -0,0 +1,409 @@
+---
+eip: 7093
+title: Social Recovery Interface
+description: Interfaces for social recovery account supporting various guardian types and customizable recovery policies.
+author: John Zhang (@johnz1019), Davis Xiang (@xcshuan), Kyle Xu (@kylexyxu), George Zhang (@odysseus0)
+discussions-to: https://ethereum-magicians.org/t/eip-social-recovery-interface/14494
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-05-29
+---
+
+
+
+## Abstract
+
+This ERC proposes a standard interface for social recovery of smart contract accounts. It separates identity and policy verification from the recovery process, allowing more types of Guardians than just on-chain accounts. It also lets users customize recovery policies without changing the account’s smart contract.
```

Comment:
It'd be nice to expand the definition of "Guardian" here in the abstract, maybe something like:

```suggestion
This ERC proposes a standard interface for social recovery of smart contract accounts. It separates identity and policy verification from the recovery process, allowing more ways to authenticate (known as Guardians) than just on-chain accounts. It also lets users customize recovery policies without changing the account’s smart contract.
```